src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 700;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+}
+
+.form {
+  margin: 20px 0;
+  padding: 0 8px;
+}
+
+.label {
+  position: relative;
+  display: block;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  padding: 0 0 0 62px;
+  line-height: 70px;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  font-family: inherit;
+  font-weight: 400;
+}
+
+.input__small {
+  height: 42px;
```

Comment:
Same here - use padding instead of hard codding height

Reply:
Done